interface/material-editor.md

```diff
@@ -0,0 +1,18 @@
+# Material Editor
+The **Material Editor** allows us to create a **Material** that gives a **Mesh** a specific visual appearance, when it comes into contact with **Light** in a **Scene**. The **Material** files have a *.incmat* as their filename extension.
```

Comment:
Materials define the shading model and parameters used when rendering 3D geometry.

Also, here is some text that I wrote for the newsletter. Maybe there is some information which can be added to the material-related documentation pages:

With the introduction of PBR (physically-based rendering) materials in Incari, 3D models can now be shaded in a much more standardized way, using HDRI environment maps to create realistic reflections and lighting behavior. In addition to the standard PBR shading model, Incari also offers a glass and car paint shader, as well as Phong and shadeless models.

Reply:
I was thinking the will be a separate documentation to discuss just **Material**s with all the above information.